README.md

```diff
@@ -83,11 +83,22 @@ this is properly imported:
 #### Routes
 Update your `config/routes.rb` to include the following:
 ```ruby
-mount AtomicCms::Engine => "/atomic_cms"
+mount AtomicCms::Engine => "/atomic_cms", as: :atomic_cms
```

Comment:
Probably. But not necessarily in the best location of a customized routes.rb file.